brayns/common/scene/Model.h

```diff
@@ -189,6 +190,27 @@ class Model
      * now on.
      */
     BRAYNS_API void addModel(ModelPtr model, const Transformations& transform);
+    
+    /**
+      Adds a SDFGeometry to the scene
+      @param materialId Material of the geometry
+      @param geom Geometry to add
+      @param neighbours List of global indices of the geometry neighbours
+      @return Global index of the geometry
+      */
+    uint64_t addSDFGeometry(const size_t materialId, const SDFGeometry& geom,
+                            const std::vector<size_t>& neighbours);
+
+
+    /** Set the list of neighbours for a SDF geometry
+      @param geometryIdx Index of the geometry
+      @param neighbourIndices Indices of the neighbours
+      */
+    void setSDFGeometryNeighbours(size_t geometryIdx,
+                                  const std::vector<size_t>& neighbourIndices);
```

Comment:
How about renaming this function to updateSDFGeometryNeighbours? They are already set in addSDFGeometry() if I'm correct.

Also, this may be a stupid question, but where do I find the geometryIdx to pass to this function?